zipline/pipeline/engine.py

```diff
@@ -32,6 +32,11 @@
 from zipline.utils.sharedoc import copydoc
 
 
+DATE_INDEX_NAME = 'dates'
+SID_INDEX_NAME = 'sid'
+PIPELINE_INDEX_NAMES = (DATE_INDEX_NAME, SID_INDEX_NAME)
```

Comment:
I'd probably put these in `zipline.pipeline.common` instead of in engine.

Reply:
Good point, moving them.